packages/headless/src/features/folding/folding-state.ts

```diff
@@ -0,0 +1,32 @@
+import {Result} from '../../api/search/search/result';
+
+export interface FoldedResult extends Result {
+  /**
+   * The children of this result sorted in the same order as the search results.
+   */
+  children: FoldedResult[];
+}
+
+export interface FoldingFields {
```

Comment:
Should this one and the following be documented too?
Otherwise all the strings look good 👍

Reply:
Ah that makes sense thanks, we can leave this as is.